### DISCUSSION TITLE
Level 4 headings - how to make them appear in contents menu?

### DISCUSSION THREAD

**Discussion by benkoshy** [retypeapp/retype#355] [Q&A] [answered]
Is it possible to present lower level headings in the contents menu (as a link)?

![image](https://user-images.githubusercontent.com/15097447/177404710-8a46c639-c282-45d3-ba53-db01b9d175ba.png)

**Comment by geoffreymcgill** [comment, marked as answer]
Hi @benkoshy. Sorry, not supported. Only H2 and H3 headings will be added to the right table of contents.
